lib/tasks/metro.rake

```diff
@@ -47,4 +47,13 @@ namespace :metro do
       )
     end
   end
+
```

Comment:
Yes, which is why I wasn't sure if it should be a task or a migration since I can't imagine needing to run it more than once. Is this the right way to handle it?